ansible_events_ui/db/models.py

```diff
@@ -163,6 +163,30 @@
     sqlalchemy.Column("playbook_id", sqlalchemy.ForeignKey("playbook.id")),
 )
 
+# RBAC
+
+roles = sqlalchemy.Table(
+    "role",
+    metadata,
+    sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True),
```

Comment:
Please use identity constraints for autoincrementing primary keys.

Reply:
This?  `sqlalchemy.Identity(always=True),`

Reply:
Yes. This constraint prevents quite common type of mistake, when database table relies on autoincrementing primary key and a record is inserted with a manually defined value of a primary key, which creates a conflict next time when autoincrement produces the value already inserted in the table.